server/items/itemsController.js

```diff
@@ -0,0 +1,139 @@
+var Item = require('./itemModel.js');
+var pg = require('pg');
+var connectionString = process.env.DATABASE_URL || 'postgres://localhost:5432/juju';
```

Comment:
Can you change this path as var connectionString = process.env.DATABASE_URL ||  config.connectionString; ?